test/notificationSpec.js

```diff
@@ -13,6 +13,8 @@ const transferExpiryMonitor = require('../src/services/transferExpiryMonitor')
 const transferDictionary = require('five-bells-shared').TransferStateDictionary
 const transferStates = transferDictionary.transferStates
 
+const dbFailureMock = require('../src/models/db/utils')
```

Comment:
'../src/models/db/utils' was mock-required by the helper. i'll add a comment, though.